src/adapters/suggest.js

```diff
@@ -214,6 +217,23 @@ export default class Suggest {
     this.autocomplete.clear();
   }
 
+  renderEmpty(query = '') {
+    if (query.trim() === '') {
+      return '';
+    }
+    return `
+      <div class="no_suggestion">
+        <div class="first_line">
+          ${_(`Impossible to find "${query}" in Qwant Maps...`)}
+        </div>
+        <div class="second_line">
+          ${_('You can add this place in OpenStreetMap so you can find it in Qwant Maps')}
+          <a href="https://learnosm.org/en/beginner/start-osm/">${_('here')}</a>.
```

Comment:
I am not convinced that this wording is appropriate. In many cases the query may refer to a place that does exist in OSM but is not indexed in our database, or under a different name/spelling, etc. Also a query may refer to a larger intention and not to a specific place.  

In my opinion this message should be less insistent.